src/helpers/helpers.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+let idUser = 0;
+
+
+const uniqueUserId = (name) => {
+  switch (name) {
+    case 'user':
+      idUser++;
+
+      return idUser;
+    case 'expenses':
+      idExpenses++;
+
+      return idExpenses;
+  }

```

Comment:
why do you need it if you already create separate func for it? and then why do you need switch case?